src/openforms/forms/admin/form.py

```diff
@@ -48,6 +48,21 @@ class FormStepInline(OrderedTabularInline):
     extra = 1
 
 
+class FormReachedSubmissionLimitListFilter(admin.SimpleListFilter):
+    title = _("has reached submission limit")
+    parameter_name = "submission_limit"
```

Comment:
Can we not use a filter that's simpler like the filters for a boolean field? it's either Submission limit reached or not.